src/renderers/WebGLRenderer.js

```diff
@@ -3351,8 +3351,8 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 		}
 
-		_gl.activeTexture( _gl.TEXTURE0 + slot );
-		_gl.bindTexture( _gl.TEXTURE_2D, texture.__webglTexture );
+		state.activateTexture( _gl.TEXTURE0 + slot );
```

Comment:
One last thing. Instead of `activateTexture` I would name it `activeTexture` so it matches the WebGL API.